pandas-stubs/core/base.pyi

```diff
@@ -22,18 +23,22 @@ from typing_extensions import Self
 
 from pandas._typing import (
     S1,
+    ArrayLike,
     AxisIndex,
     DropKeep,
     DTypeLike,
     GenericT,
     GenericT_co,
     NDFrameT,
     Scalar,
+    SequenceNotStr,
     SupportsDType,
     np_1darray,
 )
 from pandas.util._decorators import cache_readonly
 
+_ListLike: TypeAlias = ArrayLike | dict[str, np.ndarray] | SequenceNotStr[S1]
```

Comment:
not sure why this gets redefined in multiple files, but i've kept to the pattern

ok, or better to redefine in `_typing`?

Reply:
If you do want to use it here, then I think moving the definition to `typing.pyi` makes sense, but exercise care on the naming, because there is a `ListLike` in `_typing` that is used elsewhere and I'm not sure whether they should be the same or not.   Maybe the name `_ListLike` is misleading.

Might be better to do the copying like you've done here, OR have `series.pyi` import `_ListLike` from `core/base.pyi`

Reply:
> Might be better to do the copying like you've done here

sure, keeping like this for now then, thanks